src/course-home/outline-tab/widgets/ProctoringInfoPanel.jsx

```
@@ -82,6 +84,7 @@ function ProctoringInfoPanel({ courseId, username, intl }) {
             if (Object.keys(response).length > 0) {
               setShowInfoPanel(true);
             }
+            isResolved();
```

My only concern with this PR is will this code path always be hit? What if `getProctoringInfoData` errors? I don't see a `.catch` method, but it seems like in any unsuccessful call, `isResolved` will never resolve. Same with if there isn't a response for some reason. 
Is there a way we can ensure we will always resolve this since it will block the goals widget otherwise.

---

agree, i think you might be able to use a .finally( block just for this part as this approach wouldn't show the goals panel if the proctoring request fails

---

Added some tests, it turned out to be tricky because the proctoring info API returns a 404 if the course does not have proctoring info. I modified ProctoringInfoPanel to eat the error. This probably needs a better long term solution, I saw that the API threw 170K 404s into Splunk in 24 hours.